### DISCUSSION TITLE
Node Transaction Approval

### DISCUSSION THREAD

**Discussion by 0xNordian** [deso-protocol/dips#83] [Q&A]
Is there a way to set up a node to request to approve a transaction only once or every x days instead of every single transaction? 

<img width="733" alt="Screenshot 2021-07-16 at 18 12 05" src="https://user-images.githubusercontent.com/85548493/125977512-93bbfa25-f728-4cb6-b10f-0c6a177b1c5c.png">

**Comment by ItsAditya-xyz** [comment]
yes. it solely depends upon the node runner if they want one time permission. Check out @Love4Src node. if you need help try contacting Kanshi (the founder of Love4Src)

**Reply by 0xNordian** [reply]
Thanks for the feedback. 
In the love4src.com node, you need to approve every transaction, as I can see., same goes for tijn.club. I didn't found yet a node besides bitclout.com that doesn't require approval for every transaction. 

PS: Kanshi is supporting me to develop my node.

**Reply by 0xNordian** [reply]
Now I have understood, Kanshi has changed the identity address from identity.bitclout.com to indentity.love4src.com.

Question on that: Do we think this type of change might create distrust in the future between users, meaning, when there will be thousands of nodes if each node has its identity layer:

- Can this be used to harm users?
- Can this become a red flag for users?
- Should the identity layer be centralised on the bitclout.com node? (please, don't kill me for making this question and use the word centralised 😂)

I don't know if this is a naive question since I don't have a technical background, just curious about the potential impact on users behaviour. 

Thanks!

**Reply by matt-mcmarsh** [reply]
> Now I have understood, Kanshi has changed the identity address from identity.bitclout.com to indentity.love4src.com.
> 
> Question on that: Do we think this type of change might create distrust in the future between users, meaning, when there will be thousands of nodes if each node has its identity layer:
> 
> * Can this be used to harm users?
> * Can this become a red flag for users?
> * Should the identity layer be centralised on the bitclout.com node? (please, don't kill me for making this question and use the word centralised 😂)
> 
> I don't know if this is a naive question since I don't have a technical background, just curious about the potential impact on users behaviour.
> 
> Thanks!

I had this very issue of concern when I went to check out Kanshi's node
so was only going to login with a "spare" testing account
not my main account

**Reply by marnimelrose** [reply]
This sort of has to do with my suggestion that we create a "BitClout powered" logo in #94

**Comment by tijno** [comment]
@N0rdian @AdityaChaudhary0005 Ive fixed this on [my node](https://tijn.club) without running my own identity 

You dont need to go to that extent and can still use Bitclout identity.

![20210723-r5rpwBpZ](https://user-images.githubusercontent.com/69529928/126775003-a916dbba-f90f-4b15-bbde-dcca552826f7.png)

Just change these 2 lines:

https://github.com/bitclout/frontend/blob/56185cda82f48be4a59257f7b61d2a0df214110a/src/app/global-vars.service.ts#L708

and

https://github.com/bitclout/frontend/blob/56185cda82f48be4a59257f7b61d2a0df214110a/src/app/global-vars.service.ts#L719

and add the `?accessLevelRequest=3` query string param - amending [access levels as needed](https://github.com/bitclout/identity/blob/6057a56c87be6b69395d5ba0170964dc0369eecd/src/types/identity.ts#L24).

so it reads:

```js
this.identityService.launch("/log-in?accessLevelRequest=3").subscribe((res) => {
```

No more popup for posting & replies. But any coin transactions incl diamonds still require it at 3.
